### ISSUE TITLE
Playback skipping on node1

### ISSUE THREAD

**Issue by melecityjones** [DankMemer/Octave#53]
**Describe the bug**
Skipping and timing issues during playback. Has been going on for a few days at varying degrees of frequency and severity. 

**To Reproduce**
1. Playback music from shard #346 which appears to be node1 from conversation with developer mod in Discord. mod confirmed skipping present on this node. We're on US East region.

**Expected behavior**
No skipping as found on node0. 

**Screenshots**
N/A

**Additional context**
This started getting really bad Thursday 5/14/2020. A few updates were released but the skipping continued through to today Sunday 5/17/2020. It lightened up for about 15 minutes after the reboot at ~4PM EST, but skipping started again and slowly got worse over the next 30 minutes. Messaged in chat but figured I'd file a bug for history tracking.

**Comment by melecityjones** [comment]
tl;dr
If stuttering goes away again with consistent 0% packet loss on Inbound from Octave, will mark this as fixed.

Temporary Relief
Updating since a few updates have come through, about 30 minutes after the update from ~11:30AM EST today 5/20/2020 there were only 2 very small skips over 7 songs with relatively low (<25%) CPU usage and on both JVM and SYS w/ 54% RAM Usage, 2203 voice connections. 

Return of Stuttering
Unfortunately skipping has returned to occurring many times a song now at ~10:15PM EST (<25% CPU usage, 58% RAM usage, 5105 voice connections). 

Regional Cxn Impact?
To confirm, my Discord ping is 5-7ms and generally reporting healthy. During skipping reported in my initial post I was seeing 0% packet loss consistently, but during this bit of skipping I am seeing up to 3% packet loss with RTC: Inbound for Octave. We may see better results with switching regions here but alas I don't have that many perms on the server I am on.

Follow-Up Action
Thought about erasing this comment since it may be our connection to us-east661 but I'd like to confirm at another time if we get stuttering or not with 0% packet loss. Unsure what this looked like when skipping was minimal.

**Comment by Kodehawa** [comment]
What node are you on? Stuttering doesn't seem to be common on any of the nodes I'm able to test on rn.

**Comment by melecityjones** [comment]
I'm on node 3/5 from _info. I've been listening again this evening with no stuttering over 8 songs! :D Packet loss is consistently 0% on the Inbound connection again as well. I'm going to resolve this as last night's was probably from the loss. Thank you so much!
